src/components/Distributions/Distributions.jsx

```diff
@@ -82,44 +79,27 @@ const countryMmrColumns = [{
   sortFn: true,
 }];
 
-const Distributions = ({
-  data,
-  // error,
-  // loading,
-}) => (
+const getPage = (data, key) => (
   <div>
-    <Warning className={styles.Warning}>
-      {strings.distributions_warning_1}
-      <br />
-      {strings.distributions_warning_2}
-    </Warning>
-    <Tabs
-      inkBarStyle={{ backgroundColor: styles.blue }}
-      className={styles.tabs}
-    >
-      {Object.keys(data).map(key => (
-        <Tab
-          key={key}
-          label={strings[`distributions_tab_${key}`]}
-          className={styles.tab}
-        >
-          <Heading
-            title={strings[`distributions_heading_${key}`]}
-            subtitle={`
-              ${data[key].rows && abbreviateNumber(data[key].rows.map(row => row.count).reduce(sum))} ${strings.th_players}
-            `}
-            icon=" "
-            className={styles.Heading}
-          />
-          {(key === 'mmr') ?
-            <div id="mmr" />
-            : <Table data={data[key].rows} columns={countryMmrColumns} />}
-        </Tab>))
-      }
-    </Tabs>
+    <Heading
+      title={strings[`distributions_heading_${key}`]}
+      subtitle={`
+        ${data[key] && data[key].rows && abbreviateNumber(data[key].rows.map(row => row.count).reduce(sum))} ${strings.th_players}
+      `}
+      icon=" "
+      className={styles.Heading}
+    />
+    {(key === 'mmr') ?
+      <div id="mmr" />
+    : <Table data={data[key] && data[key].rows} columns={countryMmrColumns} />}
   </div>
 );
 
+const distributionsPages = [
+  { name: strings.distributions_tab_mmr, key: 'mmr', content: getPage, route: '/distributions/mmr' },
+  { name: strings.distributions_tab_country_mmr, key: 'country_mmr', content: getPage, route: '/distributions/country_mmr' },
```

Comment:
https://github.com/odota/ui/pull/349/files#diff-c47cedf24396573ea2d285c351adc067R63
U should add slash for img path
```
<img src={`/{image}`}
```